### DISCUSSION TITLE
Possibility to run Dash app in Jupyterlite?

### DISCUSSION THREAD

**Discussion by eriknl1982** [jupyterlite/jupyterlite#1339] [Q&A]
I know it's possible to run Dash apps in Jupyter. Would it also be possible to  run them in Jupyter lite?

I already tried a basic Dash app, but got this error:

RuntimeError: can't start new thread

Does that mean it's not possible at all, or is additional configuration required?

**Comment by jtpio** [comment]
Right threads are not supported yet.

It might be possible to do it with `webdash`. There is an issue tracking this, but not sure if anyone has tried already: https://github.com/ibdafna/webdash/issues/19
